v63004/u_inref/gtm8202.csh

```diff
@@ -0,0 +1,170 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+setenv gtm_test_jnl SETJNL
+setenv gtm_repl_instance mumps.repl
+setenv gtm_repl_instname INSTANCE1
+
+#setting this skips turning on replication in the passive start script
+setenv gtm_test_repl_skipsetreplic ""
+
```

Comment:
I see a lot of empty lines in between sections of the file. Can those be removed. Bloats the module unnecessarily, not to mention the reduced viewing space.